### PR TITLE
Private aggregation cloud update

### PR DESCRIPTION
**Pull request by maybellineboon**
**Prior to creating a pull request, please follow all the steps in the [contributing guide](CONTRIBUTING.md).**

# Description

Updated Private Aggregation reporting demo to dynamically specify both bucket and cloud environment.

To be able to use this, key and bucket must be specified in the publisher page. sample link: https://privacy-sandbox-demos-news.dev/?key=1234567890&env=aws

From the sample link, this will create an AWS private aggregation report with the bucket key of 1234567890.

## Related Issue

- Fixes #xxx

## Affected services

- [ ] Home
- [ X] News
- [ ] Shop
- [ ] Travel
- [ X] DSP
- [ ] SSP
- [ ] ALL

Other:

### CODE REVIEW

File: services/dsp/src/public/js/dsp.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+let cloudEnv = document.currentScript.getAttribute('cloudenv');
 window.addEventListener('load', (event) => {
-  let iframeAws = document.createElement('iframe');
-  let iframeGcp = document.createElement('iframe');
-  // let dsp = document.currentScript.getAttribute('dsp');
-  iframeAws.src = `https://${dsp}/private-aggregation-aws`;
-  iframeGcp.src = `https://${dsp}/private-aggregation-gcp`;
-  document.body.appendChild(iframeAws);
-  document.body.appendChild(iframeGcp);
+  if ((cloudEnv == 'aws' || cloudEnv == 'gcp') && bucket != 'undefined') {
```
Comment (Seburan) [inline services/dsp/src/public/js/dsp.js:5]: Just to confirm the logic : if the "bucket" query string is not set, the private-aggregation script won't be loaded. is that intended behavior ? 

in the services/dsp/src/public/js/private-aggregation-worklet.js you are setting a default bucketKey if it's not found in Shared Storage, but I wonder if the SS code will be executed if we don't meet the condition "bucket != 'undefined'" here.

**Review by Seburan — APPROVED** [review]
please review comment on the conditional statement.

File: services/dsp/src/public/js/dsp.js (hunk 1 above)
Comment (maybellineboon) [reply services/dsp/src/public/js/dsp.js]: Hi Severin,

Yup. It was supposed to be that if bucket key is not set in the url, we will pre-set the bucket key for the report. I've removed the `bucket!='undefined'` requirement.

Thanks!

Comment (Seburan) [reply services/dsp/src/public/js/dsp.js]: Thanks Maybelline. Looks good to me.

**Pull request merged**